templates/taxi/car_detail.html

```diff
@@ -19,4 +19,30 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if user in car.drivers.all %}
+  <form action="{% url 'taxi:delete-me-car' pk=car.id %}" method="post" class="link-to-page">
+  {% csrf_token %}
+    <input class="btn btn-danger" type="submit" value="Delete me from this car">
+  </form>
+  {% else %}
+  <form action="{% url 'taxi:assign-me-car' pk=car.id %}" method="post" class="link-to-page">
+  {% csrf_token %}
+    <input class="btn btn-success" type="submit" value="Assign me to this car">
+  </form>
```

Comment:
Ensure that the form action URLs `'taxi:assign-me-car'` and `'taxi:delete-me-car'` are correctly mapped to the intended views and that the view logic supports these operations.